src/components/accounts/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Box, Text, Flex } from '@blockstack/ui';
+import PlusCircleIcon from 'mdi-react/PlusCircleIcon';
+import { useHover } from 'use-events';
+import { Image } from '@components/image';
+
+const AccountAvatar = ({ username, avatar, ...rest }) => {
+  const firstLetter = username[0];
+  return (
+    <Flex
+      overflow="hidden"
+      flexShrink={0}
+      bg="#007AFF"
+      size="36px"
+      borderRadius="100%"
+      align={'center'}
```

Comment:
Why `{'center'}` here, and strings above?

Reply:
ah, when I start typing `align`, I hit tab, my IDE by default creates `align={}`, and I can start typing `cen` and it picks up the possible string values through typescript. It'd be great if I could change it to do that without the {}.